src/styles/theme.ts

```diff
@@ -29,6 +29,28 @@ const defaultTheme = createTheme({
         fontSize: '16px'
       }
     },
+    h5: {
+      fontSize: '24px',
+      fontHeight: '120%',
```

Comment:
`fontHeight` → `lineHeight` ですかね

```suggestion
      lineHeight: '120%',
```